### PR TITLE
Correct the method-based syntax for a few more gallery examples

### PR DESCRIPTION
**Pull request by joelostblom**

### CODE REVIEW

File: tests/examples_methods_syntax/isotype.py [hunk 1]
```diff
@@ -69,7 +69,7 @@
 alt.Chart(source).mark_point(filled=True, opacity=1, size=100).encode(
     alt.X('x:O').axis(None),
     alt.Y('animal:O').axis(None),
-    alt.Row('country:N').header(title=''),
+    alt.Row('country:N').title(''),
```
Comment (mattijn) [inline tests/examples_methods_syntax/isotype.py:72]:
```suggestion
    alt.Row('country:N').title(None),
```
I prefer `None` over `''`.

File: tests/examples_methods_syntax/layered_chart_with_dual_axis.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 source = data.seattle_weather()
 
 base = alt.Chart(source).encode(
-    alt.X('month(date):T').axis(title=None)
+    alt.X('month(date):T').title('')
```
Comment (mattijn) [inline tests/examples_methods_syntax/layered_chart_with_dual_axis.py:14]:
```suggestion
    alt.X('month(date):T').title(None)
```
Or there is another reason why you opt for `''`?

**Review by mattijn — COMMENTED** [review]
Two inline comments/suggestions

File: tests/examples_methods_syntax/isotype.py (hunk 1 above)
Comment (joelostblom) [reply tests/examples_methods_syntax/isotype.py:72]: This is interesting. I think `None` is more semantically correct but `''`  is more convenient to type and I remember that when I started to use Altair a few years ago, there was some context where `None` didn't work, but `''` did. It is possible that I was just doing something wrong back then, but I will see if I can find it again.

Comment (mattijn) [reply tests/examples_methods_syntax/isotype.py:72]: Might be, I can remember something similar. But I would consider that as a limitation of Vega-Lite at that moment.

**Comment by joelostblom** [comment]
@mattijn I forgot about this for some while, but I noticed it now and went ahead with your suggestion of using `None` instead of `""` as I agree that it is more idomatic.

**Review by mattijn — APPROVED** [review]

**Pull request merged**